src/modules/acbs/acbs-deal-party.service.ts

```diff
@@ -26,12 +26,16 @@ export class AcbsDealPartyService {
     const { data: dealPartiesInAcbs } = await this.acbsHttpService.get<AcbsGetDealPartyResponseDto[]>({
       path: `/Portfolio/${portfolio}/Deal/${dealIdentifier}/DealParty`,
       idToken,
-      onError: createWrapAcbsHttpErrorCallback({
-        resourceIdentifier: dealIdentifier,
-        messageForUnknownException: `Failed to get the deal investors for the deal with id ${dealIdentifier}.`,
+      onError: createWrapAcbsHttpGetErrorCallback({
+        messageForUnknownError: `Failed to get the deal investors for the deal with id ${dealIdentifier}.`,
+        knownErrors: [],
       }),
     });
 
+    if (dealPartiesInAcbs === null) {
```

Comment:
can we use for this: `!dealPartiesInAcbs`?

Reply:
It's a good question.

Currently ACBS will return exactly the string `null` (which our deserializer will convert to the value `null`) if the deal party doesn't exist, which is why I was checking for `null` exactly. We don't _expect_ it to return any of the other falsey values unless something has gone wrong in ACBS. 

We could use the falsey check instead - it's less specific but any of the other falsey values would mean we don't understand the ACBS response anyway. What do you think?

Reply:
ah! I see - that makes sense. I'm happy to leave it as it is. same for the second item below